### PR TITLE
CoinStats Production Merge

### PR DESCRIPTION
**Pull request by astroboyReloaded**
## For this project, I did the following:
- Create react app.
- Install all dependencies.
- Select API
- Set up components files and routes.
- Fetch data from [CoinGeckos free API](https://www.coingecko.com/en/api)
- Create `SearchBar` and `Conversion` filters.
- Style
- Testing:
   - Create unit tests for pure functions in your Redux code.
   - Create integration tests for your application using the React Testing Library.
- **All steps are in the commits**
- [Requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,180 @@
...
+- **Price convertion filter** to multiple currencies for each of the Top 100 coins. **Filter handles different amounts in both sides**.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- LIVE DEMO -->
 
-## Learn More
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [Live Demo Link](https://hello-microverse.netlify.app/) -->
```
Comment (AnselemOdims) [inline README.md]: - [ ] Please kindly include the actual deployment for this project. At this moment, this deployment is for the `Hello Microverse` project.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @astroboyReloaded , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Good job including the snapshot tests, but please kindly include tests for the pure functions in your Redux code.
- [ ] Also, please kindly include user interaction and integration tests using React Testing Library

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zainsadaqat — COMMENTED** [review]
# ❌ ❌ ❌Invalid Code Review Request ❌ ❌ ❌
# Hello @astroboyReloaded  👋  

## Something went wrong 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for this reason:

- ### You have submitted a project in which you received a required changes review and didn't get approval due to which you cannot use the same PR in the following weeks, but you need to use a fresh PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by astroboyReloaded** [comment]
Hi, @zainsadaqat! Thank you for taking the time to review my project.
I don't think I understand what you're asking for. This is my project's Pull Request. Yes, it did not get approved last time (which was a few weeks ago because I had to repeat the module, but it is the same project, the same review). I don't see why I would create a new PR. Do you simply want me to **create a new PR** for whatever reason?? I'd appreciate it if you could explain further... Thank you again!

**Comment by nicupop729** [comment]
Hi @astroboyReloaded,
I am Nicolae Pop, a Senior Code Reviewer.

Zain, your previous Code Reviewer, had to invalidate the review because there is a rule that if a PR is not approved, but a student requests a review in the following weeks using the same PR.

The reason behind this rule is to ensure that the first reviewer of the current week (@zainsadaqat in this case) is not bound by the previous review given weeks ago (in this case, the one given by @AnselemOdims). While reviewers are expected to provide consistent reviews, in the case of repeating a week, we want to have a completely fresh review from the first reviewer of the current week.

I hope this explanation makes sense.

Best of luck!

**Pull request closed without merging**